extension/mediator.js

```diff
@@ -152,19 +199,24 @@ class Mediator {
      * handles all communication received from the background script
      * and properly delegates the calls to the responsible methods
      */
+    // eslint-disable-next-line max-lines-per-function
     bgScriptsMessageListener(message) {
         switch (message.command) {
             case "responseMonitorTabLoad":
                 this.start(message.tabId);
                 break;
+            case "telemetryInfoLoaded":
+                this.telemetry.setBrowserEnv(message.env);
+                this.translationTelemetry.recordEnvironment(message.env);
+                this.translationTelemetry.recordVersions("0.5.0", "?", "?");
```

Comment:
is this the addon version? maybe is better to read it from the manifest.

Reply:
I wanted to do this in #42

Reply:
done